src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.stories.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import VideoSpotlightBlock from "./VideoSpotlightBlock";
+import videoSpotlightBlockImage from "./stories/VideoSpotlightBlock.svg";
+
+export default {
+  title: "Grid-Aware/VideoSpotlightBlock",
+  component: VideoSpotlightBlock,
+};
+
+const Template = ({
+  eyebrowText,
+  description,
+  button,
+  image,
+  playButtonLink,
+}) => (
+  <VideoSpotlightBlock
+    eyebrowText={eyebrowText}
+    description={description}
+    button={button}
+    image={image}
+    playButtonLink={playButtonLink}
+  />
+);
+
+export const VideoSpotlightBlockDefault = Template.bind({});
+VideoSpotlightBlockDefault.args = {
+  eyebrowText: "Our Impact",
+  description:
+    "Over 3,000 people have daily internet access in local shelters and resource centers.",
+  button: { text: "View Annual Report", internalLink: "/foo" },
+  image: {
+    url: videoSpotlightBlockImage,
+    alt: "Video spotlight of man speaking.",
```

Comment:
Just to honor him, even though it's just in a Storybook story, could you use Aaron's name here?

```suggestion
    alt: "Video spotlight of Aaron speaking.",
```

Reply:
Okay